dir.props

```diff
@@ -180,8 +179,8 @@
   </PropertyGroup>
 
   <!-- .NET Core build support -->
-  <PropertyGroup Condition="'$(NetCoreBuild)' != 'true'">
-    <TargetFrameworkVersion Condition="'$(TargetFrameworkVersion)' == ''">v4.5.1</TargetFrameworkVersion>
```

Comment:
Can you please also set TargetFrameworkVersion for the windows full framework build (NetCoreBuild == false and MonoBuild == false) to 4.6.? Ideally they the mono build and windows full framework build should be as close as possible.